Makefile

```diff
@@ -29,13 +29,12 @@ cluster/prepare:
 
 .PHONY: cluster/clean
 cluster/clean:
-	@kubectl get all -n $(NAMESPACE) --no-headers=true -o name | xargs kubectl delete -n $(NAMESPACE) || true
-	@kubectl get roles,rolebindings,serviceaccounts keycloak-operator -n $(NAMESPACE) --no-headers=true -o name | xargs kubectl delete -n $(NAMESPACE) || true
-	@kubectl get pv,pvc -n $(NAMESPACE) --no-headers=true -o name | xargs kubectl delete -n $(NAMESPACE) || true
-	# Remove all CRDS with keycloak.org in the name
-	@kubectl get crd --no-headers=true -o name | awk '/keycloak.org/{print $1}' | xargs kubectl delete || true
+	@kubectl delete -f deploy/service_account.yaml -n $(NAMESPACE) || true
+	@kubectl delete -f deploy/role_binding.yaml -n $(NAMESPACE) || true
+	@kubectl delete -f deploy/role.yaml -n $(NAMESPACE) || true
 	@kubectl delete namespace $(NAMESPACE) || true
-
+	@kubectl delete -f deploy/crds/ || true
```

Comment:
Just checking - what will happen if we leave PV/PVCs? If we invoke `make cluster/clean` and then run the Operator again, will it have an empty database (as expected)?

Reply:
The PVCs will be deleted automatically, since they are within the Namespace which is deleted. If you have a StorageClass with a ReclaimPolicy of "Delete" then the PVs will also be automatically deleted. If you have a ReclaimPolicy of "Retain", or if you create all your PVs manually then it will not automatically delete them.
I think if you don't have a StorageClass, the Operator will not work anyways since PVC creation is a bit different then.
Regarding Storage Classes with the Retain I cannot test this, but to my knowledge the PVs in a Released State will not automatically be reattached to a new PVC.
https://docs.openshift.com/container-platform/4.6/storage/understanding-persistent-storage.html#reclaiming_understanding-persistent-storage

Reply:
Ok, sounds good.